starters/qwik-graphql-tailwind/src/routes/homepage.classNames.ts

```diff
@@ -1,5 +1,5 @@
 export const container = 'text-center';
-export const h1Text = 'text-4xl font-bold';
-export const ul = 'list-inside list-disc my-8 text-xl';
-export const li = 'py-2';
-export const link = 'text-blue-500 underline italic hover:text-blue-800';
+export const header =
```

Comment:
I think we will have to move the classes out of separate files, but let's keep it in a separate PR (there should be a ticket for that).